src/celestia/qt/qtdeepskybrowser.cpp

```diff
@@ -311,6 +320,8 @@ DeepSkyBrowser::DSOTableModel::data(const QModelIndex& index, int role) const
         return QString("%L1").arg((observerPos - dso->getPosition()).norm(), 0, 'g', 6);
     case AppMagColumn:
         {
+            if (dso->getAbsoluteMagnitude() == DSO_DEFAULT_ABS_MAGNITUDE) return QString();
```

Comment:
```suggestion
            if (dso->getAbsoluteMagnitude() == DSO_DEFAULT_ABS_MAGNITUDE)
                return {};
```
please use one codestyle

Reply:
Fixed spacing but leaving QString to guarantee Qt compatibility in future and across OSs.

Reply:
Want to keep return under Qt's control. QString() and QString("") send back nullptr and empty string which are different in Qt and potentially could change. Return{} will be evaluating a QVariant function that could have issues across multiple C++ versions and Qt versions and definately across these programs when looking at three different operating systems and multiple versions of those operating systems. It could compile and still fail while running which will be very difficult to track down. I saw several reports of random issues in return{} returns, but they are few.

Looking at it, it probably would be better to use QString("") so all good returns are strings and not a null that could cause errors in the future further downstream in QAbstractTableModel. I consider the return from this if condition to be valid, and not an error.  Just not logical in this situation. Don't know what QAbstractTableModel does internally in deciding and acting on invalid responses.

Reply:
Looking at the docs, it seems that null/empty QString behaves identically except for the specific case of the `isNull` method, which they recommend against using. So we're probably ok with `QString()` here.